### ISSUE TITLE
NTR: has small molecule regulator, has small molecule activator, has small molecule inhibitor

### ISSUE THREAD

**Issue by pgaudet** [oborel/obo-relations#369]
Hello, 

To represent compound functions in GO, where a function is regulated by a chemical (and not by another activity), we need the following terms, children of 'has participant: 

has small molecule regulator 
            has small molecule activator
            has small molecule inhibitor

Thanks, Pascale

**Comment by pgaudet** [comment]
PR: https://github.com/oborel/obo-relations/pull/370

**Comment by cthoyt** [comment]
Are there ongoing discussions for adding inverse relations where the subject is the small molecule?

**Comment by pgaudet** [comment]
Yes we'd like to

**Comment by cthoyt** [comment]
Hi @pgaudet, any forwards movement on this? Is there anything I can do to help?

**Comment by pgaudet** [comment]
Reciprocal relations: 

is small molecule regulator -> inverse of  has small molecule regulator
is small molecule activator -> inverse of has small molecule activator
is small molecule inhibitor -> inverse of has small molecule inhibitor

**Comment by pgaudet** [comment]
We realized that 'is small molecule regulator' is incorrectly placed; it should be a child of 'enables' (@ukemi @cmungall can you please confirm this is right?) 

Also, I have added 'inverse of' for small molecule regulator and the children : 
- 'is small molecule regulator' inverse of 'has small molecule regulator'
- 'is small molecule inhibitor' inverse of 'has small molecule inhibitor'
- is small molecule activator' inverse of 'has small molecule activator'

Do I need to ALSO add disjoint for the pairs above? 

Thanks, Pascale

**Comment by ukemi** [comment]
I don't think it should be a child of enables. That means that the small molecule regulator would enable the MF. I don't think we want that do we?

**Comment by pgaudet** [comment]
ah yes - so it should be a child of regulates?

**Comment by ukemi** [comment]
In that case it would have to be an occurrent, wouldn't it? At least in GO, occurents regulate occurents. The small molecule is a continuant. It should be the inverse relation of has_participant, which I don't see.

**Comment by pgaudet** [comment]
After discussion with @ukemi I moved 'is small molecule regulator' under 'participated in'
